src/scripts/main.js

```diff
@@ -1,3 +1,38 @@
 'use strict';
 
 // write your code here
+const logoButton = document.querySelector('.logo');
+
+const successHandler = () => {
+  const message = document.createElement('div');
+
+  message.setAttribute('class', 'message');
+  message.textContent = 'Promise was resolved!';
+
+  document.body.append(message);
+};
+
+const errorHandler = () => {
+  const message = document.createElement('div');
+
+  message.setAttribute('class', 'message error-message');
+  message.textContent = 'Promise was rejected!';
+
+  document.body.append(message);
+};
+
+const promise1 = new Promise((resolve, reject) => {
+  logoButton.addEventListener('click', () => {
+    resolve('Promise was resolved!');
+  });
+});
```

Comment:
The promise `promise1` resolves when the logo button is clicked. Ensure that the button with the class 'logo' exists in the HTML to avoid any runtime errors.